app/package.json

```diff
@@ -49,6 +53,10 @@
     ]
   },
   "devDependencies": {
-    "@graphql-codegen/cli": "^2.6.2"
+    "@graphql-codegen/cli": "^2.6.2",
+    "concurrently": "^7.0.0",
+    "nodemon": "^2.0.15",
```

Comment:
what is nodemon and parcel used for?

Reply:
ufff.. I am actually not sure. I think I might have added nodemon when I was adding parcel, not knowing that we don't need it bc it's already there. I'm testing if removing it breaks anything

Reply:
update: nothing broke, so it's out